roles/manage-users/tasks/create_user.yml

```diff
@@ -10,6 +10,30 @@
       name: "{{ item.user_name | trim }}"
       mail: "{{ item.email | default('') }}"
 # The addition of expiration date is a request that will be submitted upstream
-#      expiration_date: "{{ item.expiration_date | default('') }}"
+#      expiration_date: "{{ this_user.expiration_date | default('') }}"
     with_items: "{{ users }}"
+    register: the_user
```

Comment:
Can we please change the variable from `the_user` to something more descriptive - i.e.: `idm_users_list`